package.json

```diff
@@ -54,19 +54,19 @@
 		"debounce-fn": "^5.0.0",
 		"delay": "^5.0.0",
 		"delegate-it": "^2.0.2",
-		"dom-chef": "^4.2.2",
+		"dom-chef": "^5.0.0",
```

Comment:
https://github.com/vadimdemedes/dom-chef/releases/tag/v5.0.0

We probably need to drop some types as well

Reply:
Heh, I see you already did. I wonder how come it works though, the new types in dom-chef don't include SVG